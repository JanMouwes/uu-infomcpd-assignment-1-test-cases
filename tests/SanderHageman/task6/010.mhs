main :: [Int]
  = let ye :: (Int -> [Int]) = recfun ye :: (Int -> [Int]) x = if x == 0 then Cons x Nil else Cons x (ye (x-1));
     in ye 5;
