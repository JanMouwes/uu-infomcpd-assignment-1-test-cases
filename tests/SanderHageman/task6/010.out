(Cons 5 (Cons 4 (Cons 3 (Cons 2 (Cons 1 (Cons 0 Nil))))))
